rsciio/mrc/_api.py

```diff
@@ -250,7 +303,9 @@ def file_reader(
     navigation_shape=None,
     distributed=False,
     chunks="auto",
-    metadata_file=None,
+    metadata_file="auto",
+    virtual_images=None,
+    external_images=None,
 ):
     """
     File reader for the MRC format for tomographic data.
```

Comment:
While you are at it (and before we forget!), update this docstring description, to mention 4D STEM?